psatoken.go

```diff
@@ -384,7 +405,9 @@ func (p *Claims) validateNonce() error {
 		return fmt.Errorf("missing mandatory nonce")
 	}
 
-	if err := isPSAHashType(*p.Nonce); err != nil {
+	nonce := ([]eat.Nonce)(*p.Nonce)[0]
```

Comment:
Could the nonce decoding use a consistent type as used as Nonces on line 54 ?

Reply:
I need the pending changes in the eat repo (specifically, the new `GetI()` interface) to deal with this in a clean way.